js_modules/dagster-ui/packages/ui-core/src/asset-graph/ContextMenuWrapper.tsx

```diff
@@ -18,14 +18,24 @@ export const ContextMenuWrapper = ({
   wrapperInnerStyles?: React.CSSProperties;
 }) => {
   const [menuVisible, setMenuVisible] = React.useState(false);
-  const [menuPosition, setMenuPosition] = React.useState<{top: number; left: number}>({
-    top: 0,
-    left: 0,
+  const [menuPosition, setMenuPosition] = React.useState<{
```

Comment:
This is a fix for right-clicking on an asset on the far right edge of the screen - the menu needs to be anchored from the other side so it's onscreen

Reply:
ooo I figured blueprint would have logic to make sure it's always visible regardless of setting, that's a bummer that it doesn't